index.html

```diff
@@ -65,9 +60,15 @@
                 ])
                 const [active, setActive] = useState(0)
 
-                const [currentContent, setCurrentContent] = useState(
-                    documents.find((d) => d.id == active).content,
```

Comment:
This line is only evaluated when the component is created. Therefore `currentContent` always referred to the first document.

Reply:
Side note on `d.id == active`:

```
 0 == "" => true
 0 === "" => false
```

(implicit type cast happening there)